cli/compose/types/types.go

```diff
@@ -92,75 +218,84 @@ func (s Services) MarshalYAML() (interface{}, error) {
 	return services, nil
 }
 
+// MarshalJSON makes Services implement json.Marshaler
+func (s Services) MarshalJSON() ([]byte, error) {
```

Comment:
Why is this required ?

Reply:
Because `Services` is a `ServiceConfig` slice and the expected output is a mapping. Same reason we have a `MarshalYAML` method for this type.